lib/espresso-runner.js

```diff
@@ -1,15 +1,17 @@
 import { JWProxy } from 'appium-base-driver';
 import { retryInterval } from 'asyncbox';
 import logger from './logger';
+import ServerBuilder from './server-builder';
 import path from 'path';
 import { fs, util, mkdirp } from 'appium-support';
 import { version } from '../../package.json'; // eslint-disable-line import/no-unresolved
 import request from 'request-promise';
 import B from 'bluebird';
 import _ from 'lodash';
+import { copyGradleProjectRecursively } from './utils';
 
 
-const TEST_APK_PATH = path.resolve(__dirname, '..', '..', 'espresso-server', 'app', 'build', 'outputs', 'apk', 'androidTest', 'debug', 'app-debug-androidTest.apk');
+const TEST_SERVER_ROOT = path.resolve(__dirname, '..', '..', 'espresso-server');
 const TEST_MANIFEST_PATH = path.resolve(__dirname, '..', '..', 'espresso-server', 'AndroidManifest-test.xml');
```

Comment:
this values could be replaces with path.resolve(TEST_SERVER_ROOT, 'AndroidManifest-test.xml')